control-plane/agents/src/bin/core/node/service.rs

```diff
@@ -119,6 +119,34 @@ impl NodeOperations for Service {
         let node = self.drain(id, label).await?;
         Ok(node)
     }
+
+    /// Apply the label to node.
+    async fn label(&self, id: NodeId, label: String) -> Result<Node, ReplyError> {
```

Comment:
what is label here?

Reply:
the key which has been passed
```
$ kubectl label node node-0-142132 A=B;C=D
node/node-0-142132 labeled
```
In the above case `A` or `C` is the label

Reply:
Changed it to `label: Option<HashMap<String, String>>`